interop/xds/server/server.go

```diff
@@ -118,9 +118,10 @@ func main() {
 	// If -secure_mode is not set, expose all services on -port with a regular
 	// gRPC server.
 	if !*secureMode {
-		lis, err := net.Listen("tcp4", fmt.Sprintf(":%d", *port))
+		addr := fmt.Sprintf(":%d", *port)
```

Comment:
nit: maybe init `addr` above this block and nix line 145?

Reply:
I wanted to preserve the "laziness" of formatting, if that's ok.